mule-packager/src/main/java/org/mule/tools/api/packager/packaging/PackagingOptions.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+/**
+ * DTO that represents the options when packaging mule applications.
+ */
+package org.mule.tools.api.packager.packaging;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PackagingOptions {
+
+  /**
+   * Whether a JAR file will be created for the classes in the app. Using this optional configuration parameter will make the
+   * generated classes to be archived into a jar file and the classes directory will then be excluded from the app.
+   *
+   * @parameter expression="${archiveClasses}" default-value="false"
+   */
+  private final boolean archiveClasses;
+
+
+  /**
+   * List of exclusion elements (having groupId and artifactId children) to exclude from the application archive.
+   *
+   * @parameter
+   * @since 3.2.0
+   */
+  private final ArrayList<Exclusion> exclusions;
+
+  /**
+   * List of inclusion elements (having groupId and artifactId children) to exclude from the application archive.
+   *
+   * @parameter
+   * @since 3.2.0
+   */
+  private final ArrayList<Inclusion> inclusions;
+
+  /**
+   * Exclude all artifacts with Mule groupIds. Default is <code>true</code>.
+   *
+   * @parameter default-value="true"
+   * @since 3.2.0
+   */
+  private final boolean excludeMuleDependencies;
+
+  /**
+   * @parameter default-value="false"
```

Comment:
Please remove or put it outside comment section

Reply:
idem